modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -101,157 +101,101 @@ -(void)setBridge:(RCTBridge *)newBridge
-        }
-        
-        // one more lockstate event comes along with lockcomplete
-        // when phone is locked with active application
-        if(wozniakConstant == STATE_LOCKED_WITH_ACTIVE_APP){
-            wozniakConstant = STATE_BACKGROUND;
-        }
-        
-    }
-}
-
 ////////////////////////////////////////////////////////////////////
 #pragma mark - startNode
 //////////////////////////////////////////////////////////////////// startNode
-RCT_EXPORT_METHOD(startNode:(RCTResponseSenderBlock)onResultCallback) {
+RCT_EXPORT_METHOD(startNode:(NSString *)configString) {
 #if DEBUG
     NSLog(@"StartNode() method called");
 #endif
-    if (!isStatusInitialized){
-        isStatusInitialized = true;
-        
-        NSError *error = nil;
-        NSURL *folderName =[[[[NSFileManager defaultManager]
-                              URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask]
-                             lastObject]
-                            URLByAppendingPathComponent:@"ethereum/testnet"];
-        
-        if (![[NSFileManager defaultManager] fileExistsAtPath:folderName.path])
-            [[NSFileManager defaultManager] createDirectoryAtPath:folderName.path withIntermediateDirectories:NO attributes:nil error:&error];
-        NSURL *flagFolderUrl = [[[[NSFileManager defaultManager]
-                                URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask]
-                               lastObject]
-                              URLByAppendingPathComponent:@"ropsten_flag"];
-        
-        if(![[NSFileManager defaultManager] fileExistsAtPath:flagFolderUrl.path]){
-            NSURL *lightChainData = [folderName URLByAppendingPathComponent:@"StatusIM/lightchaindata"];
-            [[NSFileManager defaultManager] removeItemAtPath:lightChainData.path
-                                                       error:nil];
-            NSString *content = @"";
-            NSData *fileContents = [content dataUsingEncoding:NSUTF8StringEncoding];
-            [[NSFileManager defaultManager] createDirectoryAtPath:flagFolderUrl.path
-                                      withIntermediateDirectories:NO
-                                                       attributes:nil
-                                                            error:&error];
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    NSError *error = nil;
+    NSURL *rootUrl =[[fileManager
+                      URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask]
+                     lastObject];
+    NSURL *testnetFolderName = [rootUrl URLByAppendingPathComponent:@"ethereum/testnet"];
+    
+    if (![fileManager fileExistsAtPath:testnetFolderName.path])
+        [fileManager createDirectoryAtPath:testnetFolderName.path withIntermediateDirectories:YES attributes:nil error:&error];
+    
+    NSURL *flagFolderUrl = [rootUrl URLByAppendingPathComponent:@"ropsten_flag"];
```

Comment:
Does this mean ropsten has special role? Is it a matter of it being default somehow?

Reply:
This is related to migration to ropsten network from previous testnet (don't remember the name), you can find this code in develop as well.

Reply:
ok